src/lib.rs

```diff
@@ -6,26 +6,40 @@ use crate::model::{ClusterState, NodeState, ScuttleButtMessage};
 
 // https://www.cs.cornell.edu/home/rvr/papers/flowgossip.pdf
 
+/// HashMap key for the heartbeat node value.
+pub(crate) const HEARTBEAT_KEY: &str = "heartbeat";
+
 pub struct ScuttleButt {
     max_transmitted_key_values: usize, // mtu in the paper
     self_node_id: String,
     cluster_state_map: ClusterState,
+    heartbeat: u64,
 }
 
 impl ScuttleButt {
     pub fn with_node_id(self_node_id: String) -> Self {
-        ScuttleButt {
+        let mut scuttlebutt = ScuttleButt {
             max_transmitted_key_values: 10,
+            heartbeat: 0,
             self_node_id,
             cluster_state_map: ClusterState::default(),
-        }
+        };
+
+        // Immediately mark node as alive to ensure it responds to SYNs.
+        scuttlebutt.self_node_state().set(HEARTBEAT_KEY, 0);
```

Comment:
Had to add this here because when writing my test I found out that if the node is completely empty, it will not be added to the cluster state and thus not be listed as a living node. I'd assume we want nodes to be alive from the get-go if they receive a syn before sending one on their own.

That said, as long as there are seed nodes usually a `ScuttleButt` will never be empty anyway.